### DISCUSSION TITLE
About CLA

### DISCUSSION THREAD

**Discussion by shubhanshu-coder** [oppia/oppia-android#4138] [Q&A (Other)] [answered]
Is there any CLA i didn't find that can anyone help?
type of form i g

**Comment by BenHenning** [comment]
Hi @shubhanshu-coder. Given that it seems you found & signed the CLA (per https://github.com/oppia/oppia-android/issues/3722#issuecomment-1023114174) is this still an issue that you need help with?

**Comment by MaskedCarrot** [comment, marked as answer]
Hi, there is one CLA and one contributor survey (as mentioned [here](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#onboarding-instructions)).
